ci/install_format_tools.sh

```diff
@@ -2,6 +2,6 @@
 
 set -e
 
-apt install -y clang-format-8 python3-pip golang git
+apt install -y clang-format-8 python3-pip git curl
 pip3 install cmake_format==0.6.5
-go get github.com/bazelbuild/buildtools/buildifier
+curl -o /usr/local/bin/buildifier https://github.com/bazelbuild/buildtools/releases/tag/2.2.1/buildifier
```

Comment:
This *needs* `chmod +x`

Currently the format script says "Can't find buildifier." but doesn't treat it as a fatal error. :)